src/applications/disability-benefits/2346/config/form.js

```diff
@@ -1,30 +1,37 @@
 import { VA_FORM_IDS } from 'platform/forms/constants';
-import fullSchemaMDOT from '../2346-schema.json';
-import personalInfoBox from '../components/personalInfoBox';
-import orderSupplyPageContent from '../components/orderSupplyPageContent';
-import orderAccessoriesPageContent from '../components/orderAccessoriesPageContent';
-import SelectArrayItemsBatteriesWidget from '../components/SelectArrayItemsBatteriesWidget';
-import SelectArrayItemsAccessoriesWidget from '../components/SelectArrayItemsAccessoriesWidget';
+import PersonalInfoBox from '../components/PersonalInfoBox';
 import { schemaFields } from '../constants';
 import ConfirmationPage from '../containers/ConfirmationPage';
 import IntroductionPage from '../components/IntroductionPage';
-import UIDefinitions from '../definitions/2346UI';
+import fullSchemaMDOT from '../schemas/2346-schema.json';
+import { buildAddressSchema } from '../schemas/address-schema';
+import UIDefinitions from '../schemas/definitions/2346UI';
+import fullNameUI from 'platform/forms-system/src/js/definitions/fullName';
 
-const {
-  email,
-  date,
-  gender,
-  address,
-  supplies,
-  accessories,
-  yesOrNo,
-} = fullSchemaMDOT.definitions;
+const { email, supplies } = fullSchemaMDOT.definitions;
 
-const { permAddressField, tempAddressField, emailField } = schemaFields;
+const { currentAddress, selectedAddress } = fullSchemaMDOT.properties;
 
-const { permanentAddress, temporaryAddress } = fullSchemaMDOT.properties;
+const {
+  emailField,
+  suppliesField,
+  viewAddAccessoriesField,
+  viewAddBatteriesField,
+  currentAddressField,
+  newAddressField,
+  selectedAddressField,
+} = schemaFields;
 
-const { emailUI, permAddressUI, tempAddressUI } = UIDefinitions.sharedUISchemas;
+const {
+  emailUI,
+  addAccessoriesUI,
+  addBatteriesUI,
+  batteriesUI,
+  accessoriesUI,
+  currentAddressUI,
+  newAddressUI,
+  selectedAddressUI,
+} = UIDefinitions.sharedUISchemas;
 
 const formChapters = {
```

Comment:
https://github.com/department-of-veterans-affairs/vets-website/pull/11999/files#r406809739
same here ;)